apps/voting/contracts/Voting.sol

```diff
@@ -213,6 +240,70 @@ contract Voting is IForwarder, AragonApp {
         _executeVote(_voteId);
     }
 
+    /**
+    * @notice Assign `_delegate` as the delegate for the sender
+    * @param _delegate address to delegate to
+    */
+    function setDelegate(address _delegate) external {
+        require(_delegate != address(0), ERROR_ZERO_ADDRESS_PASSED);
+        require(_delegate != msg.sender, ERROR_SELF_DELEGATE);
+
+        address prevDelegate = delegates[msg.sender].delegate;
+        require(_delegate != prevDelegate, ERROR_DELEGATE_SAME_AS_PREV);
+
+        if (prevDelegate != address(0)) {
+            _removeDelegatedAddressFor(prevDelegate, msg.sender);
+        }
+        _addDelegatedAddressFor(_delegate, msg.sender);
+    }
+
+    /**
+    * @notice Unassign `_delegate` from the sender
+    */
+    function resetDelegate() external {
```

Comment:
The naming is a bit misleading here - it's easy to think that reset is like set second time - it's better to call it `removeDelegate` or `unassignDelegate`

Reply:
Good catch, thank you. I'll come up with better names for these functions.

Reply:
Fixed in 94e327614f7ec8785b6fe9cf4805150877a679d6.